widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -577,7 +591,18 @@ public void onNewVisitorMediaState(VisitorMediaState visitorMediaState) {
     public void engagementEnded() {
         Logger.d(TAG, "engagementEndedByOperator");
         stop();
-        dialogController.showEngagementEndedDialog();
```

Comment:
Survey is optional? What will happen if engagement should end without survey

Reply:
If a survey is `null` it will show engagement end dialog.